app/styles/libs/_helpers.scss

```diff
@@ -140,7 +140,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 }
 
 .social-link-button {
-  width: 50px;
+  width: 46.77498px;
```

Comment:
https://codepen.io/danield770/pen/jBxJEN

Reply:
@maze-runnar  Sir Thank you so much for that advice, It really helped and now I think the issue is solved, You can see the changes in the below Comment, Again Thank you for helping.

Reply:
just resolve this one -
![Screenshot from 2021-03-03 18-37-17](https://user-images.githubusercontent.com/56407566/109810430-86c12e80-7c4f-11eb-8748-b399b3d77d9b.png)

Reply:
Pls, check Is this you screen size at 100%, Because in your browser above the text of date is also coming in four lines which were corrected few days to stay in 3 lines a few days back by someone.